Todo-app.html

```diff
@@ -4,6 +4,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
     <link rel="stylesheet" type="text/css" href="Todo-css.css" />
+    <script>
```

Comment:
You shouldnt use <script> and then put a jquery function to load the script inside. You just need to load the script with native html. Something like you did in the start to load jquery script, just isntead of an url you put a local path.